model/team_member_test.go

```diff
@@ -6,29 +6,27 @@ package model
 import (
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/require"
 )
 
 func TestTeamMemberJson(t *testing.T) {
 	o := TeamMember{TeamId: NewId(), UserId: NewId()}
 	json := o.ToJson()
 	ro := TeamMemberFromJson(strings.NewReader(json))
 
-	if o.TeamId != ro.TeamId {
-		t.Fatal("Ids do not match")
-	}
+	require.Equal(t, o.TeamId, ro.TeamId, "Ids do not match")
 }
 
 func TestTeamMemberIsValid(t *testing.T) {
 	o := TeamMember{}
 
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	err := o.IsValid()
+	require.Nil(t, err, "should be invalid")	
```

Comment:
I believe this require statement is the opposite of what we want. Before, we would `fatal` if `err == nil` and here we are requiring nil.

Nit: also, we could just make this one line like so:
```suggestion
	require.Error(t, o.IsValid(), "should be invalid")	
```

Reply:
Good catch. That explains why the tests were failing